src/main/java/io/nats/client/PurgeOptions.java

```diff
@@ -45,26 +45,6 @@ public String toJson() {
         return endJson(sb).toString();
     }
 
-    @Override
-    public boolean equals(Object o) {
-        if (this == o) return true;
```

Comment:
Not needed, never used the object is not compared with anything. Not sure why this was added in the first place.